lms/djangoapps/course_wiki/editors.py

```diff
@@ -25,7 +33,14 @@ def render(self, name, value, attrs=None):
         if value is None:
             value = ''
 
-        final_attrs = self.build_attrs(attrs, name=name)
+        # TODO: Remove Django 1.11 upgrade shim
+        # SHIM: Compensate for build_attrs() implementation change in 1.11
+        if django.VERSION < (1, 11):
```

Comment:
Hmm - didn't realize that this type of comparison worked, with `VERSION` having more members and Python knowing to compare elements in order.

Reply:
It normally wouldn't have occurred to me either, but that's how `django-filters` implemented the check.  I have since seen it like this in one or two other places as well.

Reply:
@bmedx @pwnage101 @macdiesel We should do our shim version-checking like the above from now on.